circle.yml

```
@@ -54,6 +54,20 @@ commands:
           name: Check codebase
           command: make check
 
+  build-module:
+    parameters:
+      module_target:
+        description: module target
+        type: string
+      variant:
+        description: debug or release
```

May be better keep worlds as it expected. [Here](https://github.com/mapbox/mapbox-navigation-android/pull/3363/files#diff-29944324a3cbf9f4bd0162dfe3975d88R355-R358) you send "Debug", but not "debug" 🤔 
```suggestion
        description: Debug or Release
```

---

it's not case sensitive. we have 
`description: debug or release` in many places inside `circle.yml`
and we pass `Debug` instead of `debug`, because `assembleDebug` is more readable

---

yeah, in some cases it's for reading gradle tasks, in some it relates to file name like _app-**release**.apk_. We should be ok, keep it as now